CONTRIBUTING.md

```diff
@@ -37,27 +97,38 @@ To work on the codebase for this project, you will need to have those dependenci
 <details>
 	<summary>Click to Expand</summary>
 
-* After forking this repository to your own github account, and cloning it to your dev environment, you can now create a new branch on your machine. It's wise to name this branch, after the issue you are trying to fix, or the feature you are trying to add.
+- After forking this repository to your own github account, and cloning it to
+  your dev environment, you can now create a new branch on your machine. It's
+  wise to name this branch, after the issue you are trying to fix, or the
+  feature you are trying to add.
 
-	```bash
-	git checkout -b creatingContributionGuide
-	```
+      	```bash
```

Comment:
For some reason, the `bash` and backticks are rendering in the markdown:

Screenshot:
<img width="486" alt="Screen Shot 2019-10-03 at 9 31 10 AM" src="https://user-images.githubusercontent.com/2008701/66136133-918e1480-e5c0-11e9-8357-0e07be77fcc8.png">

I'm not sure if this is because of the extra whitespace... We need to ensure that it doesn't actually render the backticks and the word `bash` in the output though.

Reply:
Fixed. Apologies, it was the whitespacing.